modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/dto/v1_0/converter/CTEntryDTOConverter.java

```diff
@@ -65,6 +70,57 @@ private String _getLocalizedValue(Field field, Locale locale) {
 			field.getLocalizedValues(), locale, LocaleUtil.getDefault());
 	}
 
+	private String _getStatusMessage(
+		com.liferay.change.tracking.model.CTEntry ctEntry,
+		HttpServletRequest httpServletRequest) {
+
+		if (ctEntry == null) {
+			return StringPool.BLANK;
+		}
+
+		StringBundler sb = new StringBundler(5);
+
+		if (ctEntry.getChangeType() == CTConstants.CT_CHANGE_TYPE_ADDITION) {
+			sb.append(
+				_language.get(
+					httpServletRequest,
+					CTConstants.CT_CHANGE_TYPE_LABEL_ADDITION));
+		}
+
+		if (ctEntry.getChangeType() == CTConstants.CT_CHANGE_TYPE_DELETION) {
+			sb.append(
+				_language.get(
+					httpServletRequest,
+					CTConstants.CT_CHANGE_TYPE_LABEL_DELETION));
+		}
+
+		if (ctEntry.getChangeType() ==
+				CTConstants.CT_CHANGE_TYPE_MODIFICATION) {
+
+			sb.append(
+				_language.get(
+					httpServletRequest,
+					CTConstants.CT_CHANGE_TYPE_LABEL_MODIFICATION));
+		}
```

Comment:
Use if and else-if so that when the if evaluates to true, the rest if checks can be skipped.